projects/mercury/src/metadata/common/values/ReferringValue.js

```diff
@@ -18,8 +17,10 @@ export const ReferringValue = ({property, entry}) => {
         switch (property.datatype) {
             case DATETIME_URI:
                 return formatDateTime(value.value);
-            case BOOLEAN_URI:
-                return <Switch checked={!!value.value} readOnly />;
+            case BOOLEAN_URI: {
```

Comment:
maybe there were two places, I would still make it case-insensitive

Reply:
you're right, I changed it now